llvm/include/llvm/ProfileData/InstrProfWriter.h

```diff
@@ -75,11 +75,16 @@ class InstrProfWriter {
   // deployment of newer versions of llvm-profdata.
   bool WritePrevVersion = false;
 
+  // The version we should use to write MemProf in.
+  memprof::MemProfVersion MemProfVersionRequested;
+
 public:
   InstrProfWriter(bool Sparse = false,
                   uint64_t TemporalProfTraceReservoirSize = 0,
                   uint64_t MaxTemporalProfTraceLength = 0,
-                  bool WritePrevVersion = false);
+                  bool WritePrevVersion = false,
```

Comment:
Do we need WritePrevVersion? I don't see it being used in the patch. External APIs can directly specify the version once this patch lands so I think WritePrevVersion may be dropped?

Reply:
This is not a change in this patch, and we should remove it separately.